### ISSUE TITLE
font-style="italic" doesn't work for Chinese publication

### ISSUE THREAD

**Issue by redleafnew** [Juris-M/citeproc-js#180]
I post this issue on https://github.com/citation-style-language/styles/issues/5323 and https://forums.zotero.org/discussion/88346/font-style-italic-doesnt-work-for-chinese-publication#latest

@ocouch suggest to post here.

The original contents:

I edited a csl file, and used font-style="italic" for <container-title> , the code is
<text variable="container-title" suffix=". " font-style="italic"/>,

and the English publication italicizes, however the Chinese publication remains unchanged.

the snapshot:
italic
<img width="620" alt="111126340-8f015e00-85ad-11eb-931d-79f947924537" src="https://user-images.githubusercontent.com/22135631/113506570-37389000-9578-11eb-8a25-6b518220dfdb.png">

the gist: https://gist.github.com/redleafnew/1139eb32d6204ca4eebfd2e90dd75d18

an example item:https://gist.github.com/redleafnew/9b71ab6b5577bb32a7504ee0916d9939

IEEE.csl also can be used as the example csl.

Many thanks.

**Comment by ocouch** [comment]
His copy paste hasn't come through well. I've confirmed this bug using @redleafnew 's example files.
Here's the TL:DR;

- `font-style="italic"` **does not** work when applied to the container-title variable when container-title contains only Chinese text: `<text variable="container-title" font-style="italic"/>`
- `font-style="italic"` **does work** when applied to the container-title variable when container-title contains any non-Chinese characters. `<text variable="container-title" font-style="italic"/>`
- `font-style="italic"` **does work** when applied to the title variable when title contains Chinese text. `<text variable="title" font-style="italic"/>`
- `font-style="italic"` **does work** when applied to the title variable when title contains English text. `<text variable="title" font-style="italic"/>`

Notably, it works correctly in the Zotero quick drop-down box in the MS-Word Add in (not sure if that's using a different implementation of cite-proc).

![image](https://user-images.githubusercontent.com/43231870/113534543-5be24580-9614-11eb-90ac-1331ec1e9cf7.png)

Only Chinese:
![image](https://user-images.githubusercontent.com/43231870/113534620-91872e80-9614-11eb-8b3a-db487ec1ac94.png)

A single non-Chinese character:
![image](https://user-images.githubusercontent.com/43231870/113534655-b2e81a80-9614-11eb-88c8-502f81cf6fb3.png)

Test File: https://www.dropbox.com/s/eqvihiwdbcx9ox3/Exported%20Items.json?dl=0
I modified only the publication (container-title) field of this item to test this.

**Comment by fbennett** [comment]
Yes, italic type is blocked for Chinese characters. Is this now an accepted
font style in Chinese publishing? (If possible, please provide a link to a
published sample---this would be a significant change, and would adversely
affect users that don't need it if not done carefully).

**Comment by ocouch** [comment]
Haha - "It's not a bug, it's a feature!"

I'm not familiar with the standards, just reproducing the issue. I'll leave that for @redleafnew, he maintains a repository of Chinese styles: https://github.com/redleafnew/Chinese-STD-GB-T-7714-related-csl

Making it an option to enable italics might be the way to go?

**Comment by redleafnew** [comment]
thanks @ocouch and @fbennett , I checked some Journals' and thesis requirements, which mention foreign publication title should be italic, but for Chinese one, no specification is regulated. When I prepared a project, italic publication title is required. As @ocouch mentioned, making it an option is indeed a good way to go.

**Comment by bwiernik** [comment]
> When I prepared a project, italic publication title is required

What do you mean by that? Is there a publisher or institution that is asking for italic typesetting for Chinese characters?

**Comment by redleafnew** [comment]
yes, the institution asked us for italic typesetting for Chinese publication title?

**Comment by ocouch** [comment]
@redleafnew Which institution(s), specifically?

**Comment by bwiernik** [comment]
Can you give a link to a style guide?

**Comment by redleafnew** [comment]
@ocouch Maybe a little complex, NSFC (https://isisn.nsfc.gov.cn/egrantweb/) project uses a personal publication list from ScholarMate (https://www.scholarmate.com/oauth/index?sys=SNS, which your own publication could be claimed or added, like Researchgate), this publication list is generated automatically with ScholarMate, however the publication list of the project participant have to be generated with Zotero or Endnote, and my university required me to use the same style with ScholarMate, that is, the Chinese publication title should italic.
![Snipaste_2021-04-07_08-18-12](https://user-images.githubusercontent.com/22135631/113792995-d8426900-9779-11eb-916c-d8c54a1bd433.png)

**Comment by fbennett** [comment]
So ... there is an automated tool that incorrectly italicizes Chinese text, and your institution is requiring you to mimic the formatting produced by the tool? (That's not intended as personal criticism. If that is an accurate description of where the need comes from, it's a very strange position for them to put you in, and we'll need to think about this.)

**Comment by redleafnew** [comment]
yes, this is the application scenarios, I will contact ScholarMate and inquire him if italic Chinese publication title is correct.

**Comment by redleafnew** [comment]
@fbennett It is not totally automated tool that incorrectly italicizes Chinese text. I contacted ScholarMate, he said NSFC provide him a template (p2 of the file, 
[01.参与者模板-简历2021通用版-20210104.docx](https://github.com/Juris-M/citeproc-js/files/6268860/01.-.2021.-20210104.docx)
, which require the publication title italic.

**Comment by zepinglee** [comment]
Italic or slanted fonts are never used for Chinese characters in serious publishing. Yet there are still some unskilled editors who just simply copy the style of western scripts without understanding it. I suggest a further discussion with the style provider.

**Comment by ttrepka** [comment]
We are using `citeproc-js` in our product and received a user complaint about the same thing in Korean - journal names are not in italics unless they contain a Latin character. I honestly have no idea if it's correct or not, but apparently some journals require that so it might be useful to have an option to enable italics in Chinese or Korean.

Here's a link for one such journal: http://www.koreahrd.or.kr/html/sub11.asp It is necessary to click the right tab and then look for section 6.

![image](https://user-images.githubusercontent.com/6588668/147093165-1b3cc440-8fb6-480c-80aa-9db77c53c1cf.png)
![Screenshot 2021-12-22 at 13 31 38](https://user-images.githubusercontent.com/6588668/147093305-ac93fce8-85fc-4c57-9004-f44698db7bf7.png)

More info from the user:
> Even if it is written only in Korean, the journal name needs to be written in italics. Below is a screenshot of the recently published article's reference.

![image](https://user-images.githubusercontent.com/6588668/147093405-a238d4bf-98cc-464d-82f0-bf241d7833f7.png)

**Comment by retorquere** [comment]
There's a whole bunch op places that has `a-z` regexen. The culprit is bound to be among those.

**Comment by bwiernik** [comment]
As noted above, Chinese, Japanese, and Korean typesetting does not as a rule have italics. Most CJK fonts do not even have italic glyphs, so if italic glyphs are forced they are automatically generated slanted characters, not proper italics (eg, https://catalog.monotype.com/font/microsoft-corporation/meiryo/italic). 

All of the examples I have seen of journals using italics with CJK characters have been verbatim copying rules do English Latin-character styles like APA, which were not written with non-Latin scripts in mind (and indeed APA generally calls for non-Latin titles to be translated or transliterated). So, I'm hesitant to introduce a change into citeproc-js that produces improper CJK typesetting without clear indication that this is a deliberate choice to format characters in a way that is supported by almost no fonts, rather than merely copying Western   citation styles without careful thought.

**Comment by 01keeper** [comment]
Please refer to the following manual and the Chinese book name is actually cited as italic.

https://www.tyndale.ca/sites/default/files/2020-05/CCSTStyleManual2016.pdf
